examples/kafka-hub/integration_test_suite/subscriber_service.bal

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/websub;
+
+isolated boolean subscriptionSuccessful = false;
+isolated boolean unsubscriptionSuccessful = false;
+isolated int recievedNotificationCount = 0;
+
+isolated function isSubscriptionSuccessful() returns boolean {
+    lock {
+        return subscriptionSuccessful;
```

Comment:
Return readonly values to get rid of the lock

Reply:
But the variable is `isolated` and `boolean` is a basic type. So, isn't we hv to use `lock` ?